torch/nn/modules/rnn.py

```diff
@@ -956,10 +939,8 @@ def __init__(self, input_size: int, hidden_size: int, bias: bool = True, nonline
         self.nonlinearity = nonlinearity
 
     def forward(self, input: Tensor, hx: Optional[Tensor] = None) -> Tensor:
-        self.check_forward_input(input)
```

Comment:
you are removing checks from RNNCell, yet I don't see added checks in rnn cell in RNN.cpp (only lstm_cell and gru_cell are modified

Reply:
Fixed it. I tried to simplify things at first and move checks from `rnn_tanh_cell` , `gru_cell`... directly to the struct they build like  `SimpleCell::operator()`... but  since `SimpleCell` have template parameter `cell_params` which is different for quantized/non-quantized case, it seemed impossible at the moment.